change/@office-iss-react-native-win32-2020-03-04-11-31-02-lehon-cursor.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "none",
```

Comment:
This should be patch. -- Otherwise a new version of react-native-win32 will not be published